src/pages/UserSite/HomePage/homePage.tsx

```diff
@@ -158,7 +159,7 @@ export default class HomePage extends React.Component{
     render(){
         const app: any = this;
 
-        app.createSearchRes();
+        app.createSearchRes(); // note
```

Comment:
Hadde staten i disse blitt satt med `this.setState()` (som får react til å rerendere) så hadde det blitt en evig loop. Heldigvis trenger du ikke calle disse her hvis du gjør noe ala det som står i kommentaren over :)